src/Widgets/Viewer.vala

```diff
@@ -200,6 +200,7 @@ public class ENotes.Viewer : WebKit.WebView {
         mkd.document (out result);
 
         string html = "<!doctype html><meta charset=utf-8><head>";
+        html += "<script type=\"text/javascript\" async src=\"/usr/share/notes-up/mathjax/MathJax.js?config=TeX-MML-AM_CHTML\"></script>";
```

Comment:
You should not be adding this script on every page. only when it's being used in the plugin

Reply:
Okay, but unfortunately, MathJaX seems not to be really happy when you load it in the body, and produce some weird behavior sometime especially for inline notations